db/migrate/20150819105725_add_lookup_value_match_to_host_and_hostgroup.rb

```diff
@@ -0,0 +1,23 @@
+class AddLookupValueMatchToHostAndHostgroup < ActiveRecord::Migration
+  def up
+    add_column :hosts, :lookup_value_matcher, :string
+    add_column :hostgroups, :lookup_value_matcher, :string
+
+    Host::Managed.where(:type => 'Host::Managed').find_in_batches(:batch_size => 100) do |group|
```

Comment:
not on my local DBs, but it's good practice to use batches in migrations which can iterate over large number of items.